tools/amp-validation/index.js

```diff
@@ -26,10 +24,12 @@ function checkEndpoints(devChannel) {
 }
 
 function runValidator(validator, devChannel) {
+    const isDev = process.argv[2] || false;
+    
     return endpoint => fetchPage
-        .get(endpoint)
+        .get(endpoint, isDev)
         .then(res => {
-            console.log(`Checking the AMP validity (${devChannel ? 'pre-release' : 'release'}) of the page at ${endpoint}, result is:`);
+            console.log(`Checking the AMP validity (${devChannel ? 'pre-release' : 'release'}) of the page at ${isDev ? 'localhost:9000' : ''}${endpoint}, result is:`);
```

Comment:
Is it enough to run the `article` app locally or does the amp test require another app? 
Also how do we explain testers what needs to run locally?